.github/workflows/build.yaml

```diff
@@ -20,6 +20,34 @@ on:
     - cron: '0 0 * * 1'
 
 jobs:
+  generate:
```

Comment:
nit: let's move it after "formatting" job and rename to "codegen" or "generation"